src/main/java/eu/bbmri_eric/negotiator/service/UserNotificationServiceImpl.java

```diff
@@ -33,10 +38,29 @@
   @Autowired EmailService emailService;
   @Autowired ResourceLifecycleService resourceLifecycleService;
   @Autowired TemplateEngine templateEngine;
+  @Autowired NegotiationRepository negotiationRepository;
 
   @Value("${negotiator.frontend-url}")
   private String frontendUrl;
 
+  @Value("${reminder.trigger-duration-days:P14D}")
+  private String triggerDuration;
+
+  private static LocalDateTime thresholdTime;
+
+  private static Collection<NegotiationResourceState> activeNegotiationResourceStates =
```

Comment:
why not do it the other way? nonActiveResource states are just Unavailable and Made available